pkg/cli/admin/release/image_mapper.go

```diff
@@ -213,7 +213,7 @@ func loadImageStreamTransforms(input, local *imageapi.ImageStream, allowMissingI
 	}
 
 	if kubectlVersions.Len() > 2 {
-		return nil, nil, nil, fmt.Errorf("input image stream has multiple versions defined for version kubectl for all the tags %s", strings.Join(kubectlVersions.UnsortedList(), ","))
+		klog.Infof("input image stream has multiple versions defined for version kubectl for all the tags %s", strings.Join(kubectlVersions.UnsortedList(), ","))
```

Comment:
How about doing `fmt.Fprintf(o.ErrOut, "warning:...` instead? To make the message stand out a bit more.

Reply:
Updated.